content/en/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -30,28 +30,28 @@ enabled, especially if [anonymous requests](/docs/reference/access-authn-authz/a
 or [service account tokens](/docs/reference/access-authn-authz/authentication/#service-account-tokens)
 are allowed.
 
-Nodes should be provisioned with the public root certificate for the cluster such that they can
+Nodes should be provisioned with the public root {{< glossary_tooltip text="certificate" term_id="certificate" >}} for the cluster such that they can
 connect securely to the API server along with valid client credentials. A good approach is that the
-client credentials provided to the kubelet are in the form of a client certificate. See
+client credentials provided to the {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} are in the form of a client certificate. See
 [kubelet TLS bootstrapping](/docs/reference/access-authn-authz/kubelet-tls-bootstrapping/)
-for automated provisioning of kubelet client certificates.
+for automated provisioning of {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} client certificates.
```

Comment:
I'm little concerned about the flooding of glossary tool tips.
We can safely assume that users can quickly learn a new concept.
It is probably okay to use a tooltip when something shows up for the very first time,
but turning all occurrences of the same word into tooltips is an overkill,
or even boring.

Please think twice.

Reply:
Hi @tengqm , I totally understand your point.
Having a tool that can describe the component or term seems too useful not to use. Sometimes user wants to revisit the said term and it felt it could help in that situation.
However, if it looks like flooding I'll leave some that seem necessary. Anyways, Thanks for the review !!